content/development/getting-started.html

```diff
@@ -17,7 +20,32 @@ <h3>Getting Involved</h3>
 and preferences are. Usually, by running Haiku for a while you will easily find
 out things that don't work as expected or that can be improved.
 
-<!--more-->
+<h2><a name="development_tools" title="development_tools"></a>What do I need for coding?</h2>
+<ul>
+	<li>
+		Currently, Haiku is the most optimal and frictionless development environment.
+		Linux, BSD, and OS X also work perfectly fine.
+                BeOS, Zeta, Cygwin, and Solaris are no longer actively maintained as development environments and little to no support is offered for those platforms. 
+	</li>
+	<li>
+		Subscribe to these mailing lists:
```

Comment:
I would word this in a more general way. "Get in touch with the other developers" or something. It doesn't matter if it's the mailing list, the forum, or the IRC channel. Also, the mailing lists are dead and no one replies :(